transform:
  translation: [-1.0, 1.0, 0.0]  # x, y, z 
  rotation_euler: [0.524, 0.0, 0.0]  # (roll, pitch, yaw)